src/app/tables/table.component.css

```diff
@@ -138,6 +138,12 @@
     display: none;
 }
 
+.estuary-table-header__num-elements {
+    margin-left: 36rem;
```

Comment:
Why is this margin set? In both Chrome and Firefox it causes the text to be way to the right of page and off the table. Without this margin, the text looks fine.

Reply:
It looks fine for me (it appears at the end of the table -on the right-, inside the table. I liked that was on the right.
But I don't like it much (the fact that it is a fixed margin). I can just remove it and when I'll try to use flex box we can see if it looks better on the right.